### ISSUE TITLE
INFO ALL returns an empty string.

### ISSUE THREAD

**Issue by anadion** [JohnSully/KeyDB#153]
**INFO ALL returns an empty string.**
If you have more then one node at replicaof and execute `INFO ALL`  after `CONFIG GET *` it returns an empty string.
It affects redis_exporter https://github.com/oliver006/redis_exporter/issues/326

**To Reproduce**
1. Create a 3 node active-replica cluster
```
version: '3.5'
services:
  keydb1:
    image: eqalpha/keydb
    container_name: keydb1
    entrypoint: ["keydb-server"]
    command: ["/etc/keydb/redis.conf", "--port", "6379", "--multi-master", "yes", "--active-replica", "yes","--protected-mode", "no", "--bind", "0.0.0.0", "--repl-diskless-sync", "yes", "--replicaof", "keydb2", "6379","--replicaof", "keydb3", "6379"]
  keydb2:
    image: eqalpha/keydb
    container_name: keydb2
    entrypoint: ["keydb-server"]
    command: ["/etc/keydb/redis.conf", "--port", "6379", "--multi-master", "yes", "--active-replica", "yes","--protected-mode", "no", "--bind", "0.0.0.0", "--repl-diskless-sync", "yes", "--replicaof", "keydb1", "6379","--replicaof", "keydb3", "6379"]
  keydb3:
    image: eqalpha/keydb
    container_name: keydb3
    entrypoint: ["keydb-server"]
    command: ["/etc/keydb/redis.conf", "--port", "6379", "--multi-master", "yes", "--active-replica", "yes","--protected-mode", "no", "--bind", "0.0.0.0", "--repl-diskless-sync", "yes", "--replicaof", "keydb1", "6379","--replicaof", "keydb2", "6379"]
```
2. Connect to one of them `docker exec -ti keydb1 keydb-cli`
3. Run `CONFIG GET *`
4. Run `INFO ALL`

**Comment by JohnSully** [comment]
I've got a repro

**Comment by JohnSully** [comment]
The issue isn't with the INFO command, rather config get returns a corrupt response with multimaster.  This confuses the CLI program and results in incorrect processing of later commands.

From telnet:
```
config get replicaof
*2
$9
replicaof
$14
127.0.0.1 6380
$9
replicaof
$14
127.0.0.1 6381
```

The *2 should be *4 because we return 4 strings.

**Comment by JohnSully** [comment]
Fixed in change 5aef606d

The new output is:
127.0.0.1:6379> config get replicaof
1) "replicaof"
2) "127.0.0.1 6380"
3) "replicaof"
4) "127.0.0.1 6381"
